.travis.yml

```diff
@@ -1,18 +1,16 @@
 language: java
 jdk:
   - oraclejdk8
-sudo: required
 
-jobs:
-  include:
-    - stage: build
-      script: mvn clean package
-      if: type = push AND branch != master
-    - stage: test
-      script:
-        - chmod +x install.sh && ./install.sh > /dev/null
-        - set -o pipefail
-        - mvn -P integration-test-azure clean cobertura:cobertura-integration-test | grep -v "DEBUG"
-        - bash <(curl -s https://codecov.io/bash)
-      if: type = pull_request OR (type = push AND branch = master)
+before_script:
+  - chmod +x ./src/libs/integration-test-setup.sh
+  - ./src/libs/integration-test-setup.sh > src/test/resources/application.properties
+  - cat src/test/resources/application.properties
+
+script:
+  - set -o pipefail
+  - mvn -P travis-ci-test clean cobertura:cobertura-integration-test | grep -E "INFO|Tests"
```

Comment:
Afraid the `grep -E "INFO|Tests"` here may miss some logs.

Reply:
due to the limitation of travisCI log 4MB, how about using `grep -v DEBUG` ?